.secrets.baseline

```diff
@@ -128,7 +128,8 @@
     {
       "path": "detect_secrets.filters.regex.should_exclude_file",
       "pattern": [
-        ".secrets.baseline"
+        ".secrets.baseline",
+        "src/aws-location-mcp-server/tests/test_server.py"
```

Comment:
We won't be excluding files, but lines by auditing the `.secrets.baseline`. Review the steps here: https://github.com/awslabs/mcp/blob/main/DEVELOPER_GUIDE.md#remediating-detected-secrets

Reply:
This will be fixed in the next PR

Reply:
Fixed